src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java

```diff
@@ -221,66 +221,188 @@ public void should_capture_int_by_creating_captor_with_primitive() throws Except
     }
 
     @Test
-    public void should_capture_byte_vararg_by_creating_captor_with_primitive() throws Exception {
+    public void should_capture_byte_vararg_by_creating_captor_with_primitive() {
         // given
         ArgumentCaptor<Byte> argumentCaptor = ArgumentCaptor.forClass(byte.class);
 
         // when
-        mock.varargsbyte((byte) 1, (byte) 2);
```

Comment:
Small question: can we keep the test as-is? Or could you please explain why we can no longer pass 2 bytes into the varargs method?

Reply:
Just like `any(byte.class)`, an `argumentCaptor(byte.class)` will now only match a single value passed to a varargs parameter.

Users should either pass a second argument captor, if they wish to match exactly two values passed to the varargs, or use `argumentCaptor(byte[].class)`, if they wish to match any number of values passed to the varargs.

Tests already exist for this for `String` vs `String[]`. However, I've added tests covering these two alternatives to make things clear for you. See `should_capture_byte_vararg_by_creating_captor_with_primitive_2_args` and `should_capture_byte_vararg_by_creating_captor_with_primitive_array`, respectively.

Reply:
Yeah this is all fine. I mostly wanted to know why we were changing the test here, which all makes sense.